dialogue-apache-hc4-client/src/main/java/com/palantir/dialogue/hc4/ApacheHttpClientChannels.java

```diff
@@ -219,9 +150,14 @@ private static MetricName clientPoolSizeMetricName(String clientName, String sta
         private final CloseableHttpClient client;
         private final ResponseLeakDetector leakDetector;
 
-        CloseableClient(CloseableHttpClient client, ResponseLeakDetector leakDetector) {
+        @Nullable
+        private final ExecutorService executor;
```

Comment:
possibly worth updating the toString while we're here?

Reply:
:+1: